sdcm/nemesis.py

```diff
@@ -2141,6 +2141,26 @@ def delete_half_partition(self, ks_cf):
 
         return partitions_for_delete
 
+    def delete_range_with_timestamp(self, ks_cf):
+        self.log.debug('Delete single partition range with timestamp')
+        partition_to_delete = self.choose_partitions_for_delete(partitions_amount=1, ks_cf=ks_cf)
+        if not partition_to_delete:
+            self.log.error('Not found partitions for delete')
```

Comment:
shouldn't we raise skip nemesis exception to mark it's skipped?

Reply:
Yes, we should. For now, everyone will look at the nemesis results and see there false positive `passed` instead of proper `skipped`.

Reply:
Ah, I was looking at how the other scylla-bench delete nemesis were done, will raise a skip there instead